test/Fail/Impossible.err

```diff
@@ -2,3 +2,4 @@ ret > ExitFailure 154
 out > An internal error has occurred. Please report this as a bug.
 out > Location of the error: __IMPOSSIBLE__, called at src/full/Agda/ImpossibleTest.hs:«line»:«col» in «Agda-package»:Agda.ImpossibleTest
 out >   impossibleTest, called at src/full/Agda/Syntax/Translation/ConcreteToAbstract.hs:«line»:«col» in «Agda-package»:Agda.Syntax.Translation.ConcreteToAbstract
+out >
```

Comment:
These extra lines should not be there.
I think the problem is that you are running the testsuite with Agda built with a to old GHC.  (We are official on GHC 9.12, but 9.10 could also work, see #7637.)